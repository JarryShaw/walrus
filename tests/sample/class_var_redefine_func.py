class A:
    print(foo := 233)

    def foo(self):
        print('foo called')
        return 666


print(type(A.foo))
print(A().foo())
