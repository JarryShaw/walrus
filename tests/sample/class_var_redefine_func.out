233
<class 'function'>
foo called
666
